site/docs/transform/filter.md

```diff
@@ -46,6 +46,54 @@ For example, to check if the `car_color` field's value is equal to `"red"`, we c
 {"filter": {"field": "car_color", "equal": "red"}}
 ```
 
+{:#lt-predicate}
+### Field Less Than Predicate
+
+{% include table.html props="field,lt,timeUnit" source="FieldLTPredicate" %}
+
+For example, to check if the `height` field's value is less than `180`, we can use the following filter:
+
+{: .suppress-error}
+```json
+{"filter": {"field": "height", "lt": 180}}
+```
+
+{:#lte-predicate}
+### Field Less Than or Equals Predicate
+
+{% include table.html props="field,lte,timeUnit" source="FieldLTEPredicate" %}
+
+For example, to check if the `Year` field's value is less than or equals to `"2000"`, we can use the following filter:
+
+{: .suppress-error}
+```json
+{"filter": {"timeUnit": "year", "field": "Year", "lte": "2000"}}
+```
+
+{:#gt-predicate}
+### Field Greater Than Predicate
+
+{% include table.html props="field,gt,timeUnit" source="FieldGTPredicate" %}
+
+For example, to check if the `state` field's value is greater than `"Arizona"`, we can use the following filter:
+
+{: .suppress-error}
+```json
+{"filter": {"field": "state", "gt": "Arizona"}}
```

Comment:
Why would you compared a state field to Arizona?

Reply:
Alphabetical order?   If that's the case, better note that `"B" < "a"` so users are more careful.